bitcoin/src/blockdata/transaction.rs

```diff
@@ -2013,10 +2007,18 @@ mod benches {
 
     #[bench]
     pub fn bench_transaction_deserialize(bh: &mut Bencher) {
-        let raw_tx = hex!(SOME_TX);
+        let raw_tx = <Vec<u8> as hex::FromHex>::from_hex(SOME_TX).unwrap();
```

Comment:
I don't understand this change. I think it's equivalent to the macro and should work equally well with `hex_lit`.

Reply:
`hex_lit` doesn't work in bench code for some reason. Seems the dev-dependecies aren't available.

Reply:
I suspect this would be fixed with criterion. We should probably have a "switch to criterion" tracking issue so that we notice when we hit a critical mass of pain with the built-in benchmarks and feel like doing the work to switch.

Reply:
> I suspect this would be fixed with criterion. We should probably have a "switch to criterion" tracking issue so that we notice when we hit a critical mass of pain with the built-in benchmarks and feel like doing the work to switch.

Added to https://github.com/rust-bitcoin/rust-bitcoin/issues/3333